src/metabase/api/dashboard.clj

```diff
@@ -188,6 +188,46 @@
   (update dashboard :ordered_cards add-query-average-duration-to-dashcards))
 
 
+(defn get-dashboard
```

Comment:
This isn't used outside of the namespace so it can be marked as private via `defn-`